apps/site/src/pages/account/general-panel/miscellaneous-topic.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Typography } from "@mui/material";
+
+import { Button } from "../../../components/button";
+
+export const MiscellaneousTopic = ({
```

Comment:
Not loving this name (if the header above where it is used ever changes, it won't make any sense) – how about `LinkButtonWithIntroText` or something else that describes what it is?